secant/Features/BackupFlow/Views/RecoveryPhraseBackupValidationView.swift

```diff
@@ -116,7 +116,8 @@ extension RecoveryPhraseValidationState {
     func wordsChips(
         for groupIndex: Int,
         groupSize: Int,
-        from wordGroup: RecoveryPhrase.Group
+        from wordGroup: RecoveryPhrase.Group,
+        state: RecoveryPhraseValidationState
```

Comment:
This function is already an extension on state, you don't have to pass it in you can just reference self.

Reply:
updated